### ISSUE TITLE
Rename todo items

### ISSUE THREAD

**Issue by odv** [jscraft-org/mentoring-project-2#14]
Add functionality in order to be able to rename to do items.

**Comment by serge-nikitin** [comment]
Let's do it in modal dialogs. It will be an interesting task. So, you press button "edit" and you should see something like this: https://v4-alpha.getbootstrap.com/components/modal/

But don't use any libraries for modals. Your own modal will look not so nice, but it's more interesting to understand how it works.

**Comment by serge-nikitin** [comment]
Also I think that "To do list" it's a wrong typing. It looks like "Todo list" is better, but I'm not sure.

**Comment by odv** [comment]
@serge-nikitin Doing this without a library was a very useful suggestion because I got to understand events. I did a lot of console.logs while implementing this functionality. So in the end I learned how to also close the modal if the user clicks outside of it by using event.target.

The **rename** modal button updates my localstorage but I think it will be great to also make it work by just pressing enter. It will be simple if I'll repeat the code inside my **popupSubmit.onclick** but I want to keep everything DRY.
